extensions/amp-google-document-embed/validator-amp-google-document-embed.protoascii

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-google-document-embed
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-google-document-embed"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-google-document-embed>
+  html_format: AMP
+  tag_name: "AMP-GOOGLE-DOCUMENT-EMBED"
+  requires_extension: "amp-google-document-embed"
+  attrs: { name: "alt" }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_url: {
+      allowed_protocol: "https"
+      allowed_protocol: "http"
+      allow_relative: true  # Will be set to false at a future date.
```

Comment:
will this actually be set to false at a future date? maybe we can just decide now if relatives should be allowed.

Reply:
Yeah this is cargo-culted in a number of places 
https://github.com/ampproject/amphtml/search?q=%23+Will+be+set+to+false+at+a+future+date&unscoped_q=%23+Will+be+set+to+false+at+a+future+date

Reply:
But why should we continue to do it if this component is new? I can understand it being that way for existing components.

Reply:
For this particular component relateive doesn't actually make sense since Url is passed to Google docs.

Reply:
I agree, I set it to disallow relative